src/components/icon-as-button/IconAsButton.jsx

```diff
@@ -53,6 +59,7 @@ IconAsButton.propTypes = {
   transparent: PropTypes.bool,
   active: PropTypes.bool,
   overlay: PropTypes.node,
+  href: PropTypes.string,
```

Comment:
What about interactive? Do we need anything to display this property there? :)

Reply:
good point. We need additional input in interactive.

Reply:
Aju is right, you should add this prop here: https://github.com/brainly/style-guide/blob/43292750917df41672d562f25d360df7053a4176/src/components/icon-as-button/pages/icon-as-button-interactive.jsx#L36 so that it's visible in the interactive part of the doc - https://styleguide.brainly.com/130.4.0/docs/interactive.html